### PR TITLE
Added new caption

### PR DESCRIPTION
**Pull request by i-aris**
# Resolves issues that I will create later
- [x] Modified _slides/08-i-aris.md with a caption.

### CODE REVIEW

**Comment by zzhang-dcs** [comment]
should I merge pull request, or just add comment saying looks good?

**Review by zzhang-dcs — APPROVED** [review]
looks good.

File: _slides/08-i-aris.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+my caption
```
Comment (VMonteverde) [inline _slides/08-i-aris.md:6]: change is ok

**Comment by i-aris** [comment]
> should I merge pull request, or just add comment saying looks good?

@zzhang-dcs I think pull request created is the one to merge it. Also, the owner is the one who deletes branches

**Pull request merged**